Gems/Multiplayer/Code/Source/Components/SimplePlayerSpawnerComponent.cpp

```diff
@@ -88,7 +88,17 @@ namespace Multiplayer
         }
 
         const AZ::EntityId spawnPointEntityId = m_spawnPoints[m_spawnIndex];
-        AZ::Transform spawnPointTransform;
+
+        if (!spawnPointEntityId.IsValid())
+        {
+            AZ_Assert(
```

Comment:
Looks good! I usually leave asserts for places that require a code change, so I’d make this an error. But hard breaks are pretty nice for fixing something that’ll bite us in the long run.

Reply:
Ugh, fair point. This _ought_ to be an error, not an assert. I've already passed AR though, so i'll submit with the assert, and we can revisit if/when we refactor the player spawner component.

Reply:
Well, yeah.. long run we need to refactor this component. Registration is backwards..

Reply:
and by long-run I mean we should probably do that nowish, in dev 😄